45 error Error: Command failed with exit code 128: git tag v0.0.1 -m v0.0.1
45 error fatal: tag 'v0.0.1' already exists
45 error     at makeError (C:\Code\github\lerna-learn\node_modules\.pnpm\execa@5.0.0\node_modules\execa\lib\error.js:59:11)
45 error     at handlePromise (C:\Code\github\lerna-learn\node_modules\.pnpm\execa@5.0.0\node_modules\execa\index.js:114:26)
45 error     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
45 error     at async VersionCommand.gitCommitAndTagVersion (C:\Code\github\lerna-learn\node_modules\.pnpm\lerna@8.1.8\node_modules\lerna\dist\index.js:9961:9)
45 error     at async VersionCommand.commitAndTagUpdates (C:\Code\github\lerna-learn\node_modules\.pnpm\lerna@8.1.8\node_modules\lerna\dist\index.js:9926:18)
45 error     at async Promise.all (index 0)
